jaxrs-api/src/main/java/jakarta/ws/rs/core/Link.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2011, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Sorry,  according to https://www.eclipse.org/projects/handbook/#ip-copyright-headers this should really be just
`Copyright (c) 2011`
The same with the rest.
Sorry for the confusion!

Reply:
Now is it wrong to have two years (create and modify) or is it simply no longer required? There's many files with two-year copyrights we probably don't want to update at this time.

Reply:
Apparently it is wrong to have two years  [](https://www.eclipse.org/projects/handbook/#legaldoc-faq) .  And yes, I see our code is all of the board in this respect.   I certainly wouldn't be in favor of fixing every file and I probably wouldn't have even brought it up if I hadn't requested that you make an incorrect change earlier (sorry).   @mkarg brought this up in a review of one of my PRs that was also incorrect.   And I will try to follow the rules going forward and I'll probably fix any part I touch that is incorrect.

Reply:
Ok, I just updated those two files again

Reply:
> Now is it wrong to have two years (create and modify) or is it simply no longer required? There's many files with two-year copyrights we probably don't want to update at this time.

According to the Eclipse Handbook it is not necessarily wrong or forbidden to have two years, but having the second year is not adviced anymore. Hence to not make it wrong, it is easiest to remove the second date whenever we touch a file, but there is no need to remove the second date unless the file is touched anyways.